Tensile/Source/TensileTypes.h

```diff
@@ -575,6 +581,7 @@ struct ProblemProperties
     // Constructor used to compute assertions for a specified problem size
     template <class ProblemDimsType>
     ProblemProperties(const ProblemDimsType& pdims, const ProblemType* props)
```

Comment:
cppcheck: Member variable 'ProblemProperties::_db' is not initialized in the constructor.